data-platform/data-science-and-ml/oracle-machine-learning/code-assets/oml-create.machine.learning.models.json

```diff
@@ -0,0 +1,1147 @@
+﻿{
+  "paragraphs": [
+    {
+      "text": "%md\n<div style=\"padding:8px; background-color:#ccff99; font-family:Calibri, sans-serif; font-size:32px; font-weight:bold\">\n&nbsp;&sect;&nbsp;&nbsp;Initialize</div>",
+      "user": "VALENTIN",
```

Comment:
User should not be a generic username. Do not use "VALENTIN". Many locations in this file.